code/script.js

```diff
@@ -13,35 +16,133 @@ const showMessage = (message, sender) => {
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/smileyface.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/Food4cure.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  // This little thing makes the chat scroll to the last message 
+  //when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Starts here 
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
+  showMessage(`Hi there, What's your name?`, 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
+//Question 1
+const handleNameInput = event => {
+  event.preventDefault()
+  const userName = nameInput.value
+  nameInput.value = ''
+  showMessage(userName, 'user')
+  setTimeout(() => askForKnowledge(userName), 1000)
+}
+
+//Question 2
+const askForKnowledge = userName => {
+  showMessage(`Lovely to meet you ${userName}! What would you like to learn about today?`, 'bot')
+
+  inputWrapper.innerHTML = `
+  <button id='zincButton'>Zinc</button>
+  <button id='proteinButton'>Protein</button>
+  <button id='vitaminDButton'>VitaminD</button>
+`
+document
+    .getElementById('zincButton')
+    .addEventListener('click', () => {
+      showMessage('zinc', 'user')
+      setTimeout(() => askForThings('zinc'), 1000)
+    })
+
+document
+    .getElementById('proteinButton')
+    .addEventListener('click', () => {
+      showMessage('protein', 'user')
+      setTimeout(() => askForThings('protein'), 1000)
+    })
+document
+    .getElementById('vitaminDButton')
+    .addEventListener('click', () => {
+      showMessage('vitaminD', 'user')
+      setTimeout(() => askForThings('vitaminD'), 1000)
+    })
+
+}
+
+//Question 3
+const askForThings = information => {
+  showMessage(`Great`, 'bot')
+  
+  if (information === "zinc") {
+    showMessage(`Zinc as a supplement if taken at onset of the first symptoms of respiratory 
+    viral diseases may half its symptoms and duration.
+    Best doses to be effective? Between 50 and 100 mg per day, 
+    to be taken after a main meal to avoid stomach upset.`, 'bot')
```

Comment:
Since these messages are quite long, you have to scroll up to read the beginning of them. What do you think about dividing them into several shorter messages and using `setTimeout` to display them one by one?

Reply:
This maybe only applies to the Protein and Vitamin D 😇